WalletWasabi.Packager/Program.cs

```diff
@@ -294,18 +294,19 @@ private static async Task PublishAsync()
 				// Delete unused executables.
 				File.Delete(Path.Combine(currentBinDistDirectory, "WalletWasabi.Fluent"));
 			}
+
 			File.Move(oldExecutablePath, newExecutablePath);
 
+			// IF IT'S IN ONLYBINARIES MODE DON'T DO ANYTHING FANCY PACKAGING AFTER THIS!!!
+			if (OnlyBinaries)
+			{
+				continue; // In Windows build at this moment it does not matter though.
```

Comment:
Maybe remove this comment?

```suggestion
				continue;
```